docs/www/configuration.md

```diff
@@ -12,102 +12,406 @@ The `redpanda` section contains all the runtime configuration, such as the
 cluster member IPs, the node ID, data directory, and so on. The `rpk` section
 contains configuration related to tuning the machine that redpanda will run on.
 
-Here’s a sample of what the config file looks like.
+Here’s a sample of the config. Only include the sections that you want to customize.
 
 ```yaml
 # organization and cluster_id help Vectorized identify your system.
 organization: ""
 cluster_id: ""
 
-redpanda:
-  # The directory where the data will be stored. It must reside on an XFS
-  # partition.
-  data_directory: "/var/lib/redpanda/data"
-
-  # An integer to identify the current node. Must be unique within the cluster.
-  node_id: 1
-
-  # The inter-node RPC server config.
-  rpc_server:
-    address: "0.0.0.0"
-    port: 33145
-
-  # The Kafka API config.
-  kafka_api:
-    address: "0.0.0.0"
-    port: 9092
-
-  # A list of `host` objects, holding known nodes' configuration. If empty, the
-  # node will be considered a root node and become available upon startup.
-  seed_servers:
-    - node_id: 1
-      host:
-        address: "0.0.0.0"
-        port: 33145
-
-  # The admin API configuration
-  admin:
-    address: "0.0.0.0"
-    port: 9644
-
-  # Target quota byte rate (bytes/ sec) - 64MB default
-  target_quota_byte_rate: 64000000
-
-  # How often (in ms) the background compaction is triggered
-  log_compaction_interval: 600000
-
-  # Max bytes per partition on disk before triggering a compaction
-  retention_bytes: null
-
-  # delete segments older than this - default 1 week
-  delete_retention_ms: 604800000000,
-
 rpk:
```

Comment:
It might just be me, but having the `rpk` config before `redpanda` config feels contrary to the natural order of things.